src/ONYXKEYS.js

```diff
@@ -9,6 +9,12 @@ export default {
     // which tab is the leader, and which ones are the followers
     ACTIVE_CLIENTS: 'activeClients2',
 
+    // Holds data related to application view state, rather than the underlying data.
```

Comment:
I'm not sure I understand the difference. What makes this data different from any other kind of data?

Reply:
Out of date, resolving.